rerun-io/rerun
# Fix camera tracking delay/flicker

Fixes #2201 
* #2201
 
The issue is about the lines flickering in and out and the proposed solution was to not render the lines. But turns out the image itself was also flickering when I tested with structure_from_motion sample. Turned out we didn't accurately track the camera, doing so works perfectly fine and doesn't need special handling on the frustum lines.

Fixes #2660 
* #2660 

I thought this was related (because I thought above would be about hiding lines), but now this is a regression fix that is just piggy backing here. PR description is about the thing that was actually broken in a release (but presumably worked before)

Test it yourself:
* got to https://demo.rerun.io/pr/2667
* click camera - observe the frustum lines are correct now
* double click the camera to focus on it and play - no flickering should occur
* try the same while the camera is already moving


### What

### Checklist
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/2667) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/2667)
- [Docs preview](https://rerun.io/preview/pr%3Aandreas%2Ffix-frustum-line-issues/docs)
- [Examples preview](https://rerun.io/preview/pr%3Aandreas%2Ffix-frustum-line-issues/examples)
